components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/AdminAdvisoryDataRetrievalClient.java

```diff
@@ -53,14 +55,16 @@ public JSONObject getAdminAdvisoryBannerData(String tenant) throws AdminAdvisory
 
             String uri = getAdminAdvisoryBannerEndpoint(tenant);
             HttpGet request = new HttpGet(uri);
-            JSONObject jsonResponse = new JSONObject();
 
             try (CloseableHttpResponse response = httpclient.execute(request)) {
                 if (response.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
-                    jsonResponse = new JSONObject(new JSONTokener(new InputStreamReader(response
+                    return new JSONObject(new JSONTokener(new InputStreamReader(response
                             .getEntity().getContent())));
                 }
-                return jsonResponse;
+                JSONObject defaultBanner = new JSONObject();
+                defaultBanner.put("enableBanner", false);
```

Comment:
Shall we introduce constants for `enableBanner` and `bannerContent` Strings?

Reply:
Fixed by https://github.com/wso2/carbon-identity-framework/pull/4767